index.html

```diff
@@ -9790,7 +9811,7 @@ <h4 id="Beast Shape - Small Animal">Beast Shape - Small Animal</h4>
                <td>Poison Strike
                </td>
                <td>Make a regular melee attack against a target. On a successful hit do
-                  Medium+DEX poison damage and <a class="condition" data-tooltip-container>hobble</a>. 2 FORT to resist <a class="condition" data-tooltip-container>hobble</a>
+                  Light+DEX poison damage and <a class="condition" data-tooltip-container>hobble</a>. 2 FORT to resist <a class="condition" data-tooltip-container>hobble</a>
```

Comment:
This isn't ok, there should not be a consistent source of 2 resistance drain added to a basic attack. Even a 1 resistance drain - we don't have it anywhere else. Even though it's just a hobble we've stated that enemies must spend to try and resist if they can so this is too much.